src/config/env/env.ts

```diff
@@ -9,6 +9,7 @@ export const defaults = {
     ORIGIN: 'localhost',
     DB_URL: 'mongodb://localhost:27017',
     AMQP_URL: 'amqp://rabbitmq:5672',
+    PROPUBLICA_API_KEY: process.env.PROPUBLICA_API_KEY,
```

Comment:
Put `invalid_api_key` instead of using process.env here since these are run time default values that do not depend on the file.